rebasehelper/patch_helper.py

```diff
@@ -107,14 +107,13 @@ def apply_patch(repo, patch_object):
             repo.index.commit('Patch: {0}'.format(os.path.basename(patch_name)), skip_hooks=True)
 
     @classmethod
-    def _update_deleted_patches(cls, deleted_patches, inapplicable_patches):
+    def _update_deleted_patches(cls, inapplicable_patches):
```

Comment:
How does this commit fix anything? It simply removes obviously unnecessary code.

Reply:
And the commit message definitely doesn't help. I think it would be best to merge this with those `base_name` => `patch_name` fixes and describe the reason behind that in commit message.